cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/web/WhoAmIServlet.java

```diff
@@ -223,7 +223,7 @@ void redirect(HttpServletResponse response, Principal principal, AuthMethod auth
 
         log.debug("redirecting to " + redirectURI.toASCIIString());
 
-        response.sendRedirect(redirectURI.getPath() + "?" + redirectURI.getQuery());
+        response.sendRedirect(redirectURI.toASCIIString());
```

Comment:
The Java integration tests were failing because of this.  I assume there is a header required to make it work that `cURL` and the browser send, but I'm not sure.  Do we keep this change?